### PR TITLE
Release 3.11.0

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Release 3.11.0
| Type?         | 
| BC breaks?    | 
| Deprecations? | 
| Fixed ticket? | 
| How to test?  |

- - -
This change is [Reviewable](https://reviewable.io/reviews/PrestaShop/ps_facetedsearch/785)

### CODE REVIEW

File: upgrade/upgrade-3.11.0.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+function upgrade_module_3_11_0($module)
+{
+    // Get all filter templates
+    $filterTemplates = Db::getInstance()->executeS(
+        'SELECT * FROM ' . _DB_PREFIX_ . 'layered_filter'
+    );
+
+    // Add controller info to each of the configuration
+    foreach ($filterTemplates as $template) {
```
Comment (kpodemski) [inline upgrade/upgrade-3.11.0.php:32]: @Hlavtox I think we should check if there's something to loop on

Comment (Hlavtox) [reply upgrade/upgrade-3.11.0.php:32]: Yeah wait

Comment (Hlavtox) [reply upgrade/upgrade-3.11.0.php]: Fixed

**Review by kpodemski — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @Hlavtox ,

Tested on 1.7.8.8, 8.0.1 and 8.1.0.

Upgrade to 3.11.0 is successful.

Tested the following :
https://github.com/PrestaShop/ps_facetedsearch/pull/757 ✅ 
Screenshot 2023-02-03 at 14 24 41

https://github.com/PrestaShop/ps_facetedsearch/pull/783 ✅ 

Screenshot 2023-02-03 at 14 26 07

And overall check of the module on FO.

It is QA ✅

**Comment by nicosomb** [comment]
Thank you @Hlavtox and @florine2623 !

**Pull request merged**